explorer/main.go

```diff
@@ -229,64 +231,18 @@ func BasicAuth(h spin.RouterHandle, requiredUser, requiredPassword string) spin.
 }
 
 // CredsOrDefault checks the KV store for a `credentials` key, and expects
-// the value to be `username:password`. If a value is not found, this function
-// will generate a random pair and log it once.
+// the value to be `username:password`. If a value is not found, it will error.
 func GetCredentials() (string, string, error) {
-	store, err := kv.Open("default")
-	if err != nil {
-		log.Printf("ERROR: cannot open store: %v", err)
-		return "", "", fmt.Errorf("error opening store: %v", err)
-	}
+	creds, err := variables.Get(KV_STORE_CREDENTIALS_KEY)
 
-	exists, err := kv.Exists(store, KV_STORE_CREDENTIALS_KEY)
 	if err != nil {
-		return "", "", fmt.Errorf("cannot check if credentials exists: %v", err)
-	}
-
-	if !exists {
-		defaultUser, err := GenerateRandomString(10)
-		if err != nil {
-			return "", "", fmt.Errorf("failed to generate random string for user: %v", err)
-		}
-		defaultPassword, err := GenerateRandomString(30)
-		if err != nil {
-			return "", "", fmt.Errorf("failed to generate random string for password: %v", err)
-		}
-
-		kv.Set(store, KV_STORE_CREDENTIALS_KEY, []byte(defaultUser+":"+defaultPassword))
-
-		log.Printf("Default user: %v", defaultUser)
-		log.Printf("Default password: %v", defaultPassword)
-		log.Printf("This is a randomly generated username and password pair. To change it, please add a `credentials` key in the default store with the value `username:password`. If you delete the credential pair, the next request will generate a new random set.")
-
-		return defaultUser, defaultPassword, nil
-	}
-
-	creds, err := kv.Get(store, KV_STORE_CREDENTIALS_KEY)
-	if err != nil {
-		return "", "", fmt.Errorf("cannot get credentials pair from store: %v", err)
+		fmt.Printf("ERROR: cannot get credentials pair from config store: %v\n", err)
+		fmt.Printf("Set the 'kv_explorer_user' and 'kv_explorer_password' variables using 'spin cloud variables set'.\n")
```

Comment:
nit: could this potentially be confusing to local spin users?

Reply:
These are logs. They are displayed in the cloud logs not in the UI. The error is caught in the front end and the associated message is here: https://github.com/kate-goldenring/spin-kv-explorer/blob/d8b666365e72552dc3734f3f5bba48f6461810a4/explorer/index.html#L314-L315:

```
alert(`Access denied to store with label "${storeLabel}." Ensure the app is linked to the store with this label and that the label has been specified in the Spin manifest (spin.toml).`);
```

Reply:
They'll be printed to the local console under `spin up` though won't they?

Reply:
As @itowlson points out, that is part of the local spin I was trying to point at.

Reply:
I see -- that's a great point. I was hyper focused on the Cloud scenario. Are you suggesting we remove this line? :
```
fmt.Printf("Set the 'kv_explorer_user' and 'kv_explorer_password' variables using 'spin cloud variables set'.\n")
```

Reply:
Suggestion:

"The [correct variables] for the application are not set. You can disable authentication using the [skip auth variable], or set them in you application using [link to setting app variables for all scenarios running Spin]"?

Reply:
I updated the log to give directions for both local and deployed apps:
```
The 'kv_explorer_user' and 'kv_explorer_password' variables for the application are not set. For deployed applications, set the variables using 'spin cloud variables set'. For local development, you can disable authentication using '--env SPIN_APP_KV_SKIP_AUTH=1' or set them in the application using runtime configuration (https://developer.fermyon.com/spin/v2/dynamic-configuration#application-variables-runtime-configuration)
```